EnglishFilter/sections/foreign.txt

```diff
@@ -2457,6 +2457,8 @@ bb.is$$div[class="banner"]
 ratopati.com##div[style*="max-height:150px;overflow:hidden;"]
 ! https://github.com/AdguardTeam/AdguardFilters/issues/34513
 jagran.com##.article-ctn
+! https://github.com/AdguardTeam/AdguardFilters/issues/41087
+timesofindia.indiatimes.com##div[data-primead="articleshow665X100"]
```

Comment:
Too specific, please check if ##div[data-primead] breaks something.

Reply:
> please check if ##div[data-primead] breaks something.

Looks fine on my end. Also added rule to hide third-party news.